src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -501,6 +501,45 @@ public object[] Geometry()
             return converted.ToArray();
         }
 
+        /// <summary>
+        /// Gets all elements hosted by the supplied element
+        /// </summary>
+        /// <param name="includeOpenings">Include rectangular openings in output</param>
+        /// <param name="includeShadows">Include shadows in output</param>
+        /// <param name="includeEmbeddedWalls">Include embedded walls in output</param>
+        /// <param name="includeSharedEmbeddedInserts">Include shared embedded elements in output</param>
+        /// <returns>Hosted Elements</returns>
+        public List<Element> GetHostedElements(bool includeOpenings = false,
+                                           bool includeShadows = false,
+                                           bool includeEmbeddedWalls = false,
+                                           bool includeSharedEmbeddedInserts = false)
```

Comment:
in terms of style, we should prefer indentation for all parameters on next line, VS2019 has a refactor for this